services/mailchimp/config.js

```diff
@@ -1,5 +1,5 @@
 const groups = {
-  techPageSignup: '17dfd6ce16',
+  techPageSignup: process.env.ENVIRONMENT_NAME === 'live' ? '17dfd6ce16' : 'd2e5a52843',
```

Comment:
Not sure if this is the correct env var we should be using. Like is this actually an env var in the lamda process? May be worth double checking how this works. I have some ideas around some alternative approaches for the mailchimp group mapping. Let have a chat when you get around to it.